Build/Scripts/Bootstrap.js

```diff
@@ -27,6 +27,8 @@ function printHelp() {
     console.log("--with-android  : Build with Android platform support");
     console.log("--with-ios      : Build with iOS platform support");
     console.log("--with-web      : Build with Web platform support");
+	console.log("--opengl        : Use OpenGL instead of DirectX 9 (only affects Windows builds)");
+	console.log("--d3d11         : Use DirectX 11 instead of DirectX 9 (only affects Windows builds)");
     console.log("--debug         : Build debug version of the editor and associated platform runtimes");
```

Comment:
White space issues, it helps to use Atom/VSCode with the "Show Invisibles" option enabled, I always have this on now. 

How about checking for Windows platform and logging "Enable OpenGL Renderer" and "Enable Direct3D  11 Renderer", otherwise "Enable OpenGL Renderer (Windows)" would work